docs/modules/configuration/pages/dynamic-config-java.adoc

```diff
@@ -0,0 +1,121 @@
+client.getConfig().addMapConfig(noBackupsMap);
+----
+====
+
+++++
+</br>
+++++
+
+.Example of loading from a configuration file
+[%collapsible]
+====
+.Member API
+[source,java]
+----
+Config config = Config.loadFromFile(/path/to/config/file);
+HazelcastInstance instance = Hazelcast.newHazelcastInstance(config);
+MapConfig noBackupsMap = new MapConfig("dont-backup").setBackupCount(0);
+instance.getConfig().addMapConfig(noBackupsMap);
+----
+
+.Java Client
+[source,java]
+----
+HazelcastInstance client = HazelcastClient.newHazelcastClient();
+MapConfig noBackupsMap = new MapConfig("dont-backup");
+noBackupsMap.setBackupCount(0);
+client.getConfig().addMapConfig(noBackupsMap);
+----
+====
+
+
+When the member receives the request, it will check if the configuration already exists and propagate any new configuration across the rest of the cluster.
+
+include::partial$dyn-config-partial-propagation.adoc[leveloffset=+1]
+
+[[handling-configuration-conflicts]]
+== Handling Configuration Conflicts
+
+You can add only new dynamic configuration. You cannot use dynamic configuration to override existing configuration at runtime.
+
+Attempting to add a dynamic configuration, when a configuration for
+the same element already exists, throws `InvalidConfigurationException`.
+
+For example,
+assuming you start a member with the following fragment in `hazelcast.xml` configuration:
+
+[source,xml]
+----
+<hazelcast>
+    ...
+    <map name="sessions">
+        ...
+    </map>
+    ...
+</hazelcast>
+----
+
+Then, adding a dynamic configuration for a map with the name `sessions` throws `InvalidConfigurationException`:
+
+[source,java]
+----
+HazelcastInstance instance = Hazelcast.newHazelcastInstance();
+
+MapConfig sessionsMapConfig = new MapConfig("sessions");
+
+// this will throw ConfigurationException:
+instance.getConfig().addMapConfig(sessionsMapConfig);
+----
+
+When attempting to add dynamic configuration for an element for which dynamic
+configuration has already been added, then if a configuration conflict is detected `InvalidConfigurationException` is thrown:
+
+[source,java]
+----
+HazelcastInstance instance = Hazelcast.newHazelcastInstance();
+
+MapConfig sessionsMapConfig = new MapConfig("sessions").setBackupCount(0);
+instance.getConfig().addMapConfig(sessionsMapConfig);
+
+MapConfig sessionsWithBackup = new MapConfig("sessions").setBackupCount(1);
+// throws ConfigurationException because the new MapConfig conflicts with existing one
+instance.getConfig().addMapConfig(sessionsWithBackup);
+
+MapConfig sessionsWithoutBackup = new MapConfig("sessions").setBackupCount(0);
+// does not throw exception: new dynamic config is equal to existing dynamic config of same name
+instance.getConfig().addMapConfig(sessionsWithoutBackup);
+----
+
+To ignore conflicts between dynamic configurations, set the xref:ROOT:system-properties.adoc#ignore-dynamic-conf-conflicts[`hazelcast.dynamicconfig.ignore.conflicts`] system property to `true`.
```

Comment:
@ramizdundar Isn't this the property for buggy config? I don't think we should publicly document this.

Reply:
This is how it was previously, it's not my decision to remove it. But if it were, I would remove it.

Reply:
If the only reason this exists is a workaround for buggy configs, I think we should remove it. It's documented in the system properties if support needs it. What do you think @Holmistr ?

Reply:
Ah I thought this was added by you, no worries then.